package.json

```diff
@@ -69,14 +69,14 @@
     "lint:fix": "eslint --fix \"**/*.{js,ts}\" && solhint --fix src/**/*.sol",
     "format": "prettier --check \"**/*.{ts,js,sol}\"",
     "format:fix": "prettier --write \"**/*.{ts,js,sol}\"",
-    "compile": "make build",
-    "test": "make test",
+    "compile": "forge build",
+    "test": "forge test -vvv",
```

Comment:
i would vote to leave out the `-vvv` flag since it shows console.log statements and when running the whole test suite we dont really need to see them in my opinion.
Is there a specific reason why you choose to add it?

Reply:
I just like to see everything. Can remove.